pkg/handler/repositories_test.go

```diff
@@ -176,6 +176,14 @@ func mockTaskClientEnqueueAddUploads(repoSuite *ReposSuite, repo api.RepositoryR
 		repo.OrgID,
 		repo.RepositoryUUID,
 	).Return(nil)
+	repoSuite.tcMock.On("Enqueue", queue.Task{
+		Typename:     config.UpdateLatestSnapshotTask,
+		Payload:      tasks.UpdateLatestSnapshotPayload{RepositoryConfigUUID: repo.UUID},
+		Dependencies: []uuid.UUID{uuid.Nil},
```

Comment:
I think the add uploads task should be a dependency, similar to here: https://github.com/content-services/content-sources-backend/blob/main/pkg/handler/repositories.go#L954

Reply:
ahh i see, just pushed that up. is that what you mean?

Reply:
oh sorry I didn't realize I was looking at a test :man_facepalming:. What you had was okay!

Reply:
Either way is fine

Reply:
ok lol i was a little confused, i'll just keep it as it is now then :)